src/components/Book.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Vr from './Vr';
+
+export default function Book() {
```

Comment:
- [ ] It will be nice if this component receives a prop (title and author) then render them.

See example:
```js
export default function Book({ prop }) {
    const { title, author } = prop;    
    
    return (
     // your jsx goes in here
   )
};
```